app/javascript/packs/SidebarWidget.jsx

```diff
@@ -6,6 +6,8 @@ HTMLDocument.prototype.ready = new Promise(resolve => {
     return resolve();
   }
   document.addEventListener('DOMContentLoaded', () => resolve());
+
```

Comment:
`HTMLDocument.prototype.ready` is defined in several files. A future refactor will sort this out as it's not directly related to the purpose of this PR.